src/CMakeLists.txt

```diff
@@ -491,10 +491,17 @@ if(WITH_BLUESTORE_PMEM OR WITH_RBD_RWL)
     if(WITH_RBD_RWL)
       list(APPEND pmdk_COMPONENTS pmemobj)
     endif()
-    find_package(pmdk 1.8 REQUIRED COMPONENTS ${pmdk_COMPONENTS})
+    find_package(pmdk 1.8 REQUIRED
+      COMPONENTS ${pmdk_COMPONENTS})
   else()
     include(Buildpmdk)
-    build_pmdk()
+    set(enable_ndctl OFF)
+    if(WITH_BLUESTORE_PMEM)
+      set(enable_ndctl ON)
+      find_package(ndctl 63 REQUIRED)
+      find_package(daxctl 63 REQUIRED)
```

Comment:
Do we need to ensure that `ndctl 63` and `daxctl 63` checks are satisfied when building against system-provided PMDK as well?  Or does `pmdk 1.8` check already cover that?

Reply:
Sorry, not quite understand. If it is the `pmdk 1.8` of the system, there is no need to build the `pmdk`.  And the `pmdk` of the system comes with `ndctl`. However, if we build `pmdk`, the `ndctl` version is required to be `>= 63`. see https://github.com/pmem/pmdk
```
Linux
You will need to install the following required packages on the build system:

autoconf
pkg-config
libndctl-devel (v63 or later)1
libdaxctl-devel (v63 or later)
```